doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -422,6 +426,17 @@ The EVP_PKEY_CTX_get0_ecdh_kdf_ukm() macro gets the user key material for B<ctx>
 The return value is the user key material length. The resulting pointer is owned
 by the library and should not be freed by the caller.
 
+=head2 ECDSA K (nonce) type parameters
+
+The EVP_PKEY_CTX_set_ecdsa_nonce_type() macro sets the ECDSA K (nonce) type to
+B<nonce_type> for ECDSA signing. Possible values are B<EVP_PKEY_ECDSA_NONCE_RANDOM>
```

Comment:
Use items here for these values (search in the pod files for examples) and describe both cases, and which one is the default.